README.md

```diff
@@ -108,66 +115,78 @@ For more information about configuring your Cloud Pak for Data cluster, see http
 
    ![Example app](./docs/images/example-app.png)
 
+For more information on how each component can be customized and configured, check out our hosted [storybook](https://watson-developer-cloud.github.io/discovery-components/storybook)
+
 ## Using Discovery Components in a React application
 
 If you don't have a React application already, start with [create react app](https://github.com/facebook/create-react-app) then modify the following in your `src/App.js`. Otherwise, you may use Discovery Components inside of any existing React component.
 
-Add the component, style, and client library to your application:
+1. Add the component, style, and client library to your application:
 
-```
-yarn add @ibm-watson/discovery-react-components @ibm-watson/discovery-styles ibm-watson
-```
+   ```
+   yarn add @ibm-watson/discovery-react-components @ibm-watson/discovery-styles ibm-watson carbon-components carbon-components-react
+   ```
 
-or
+   or
 
-```
-npm i @ibm-watson/discovery-react-components @ibm-watson/discovery-styles ibm-watson
-```
+   ```
+   npm install --save @ibm-watson/discovery-react-components @ibm-watson/discovery-styles ibm-watson carbon-components carbon-components-react
+   ```
 
-Add the `DiscoverySearch` component with corresponding `searchClient` and optionally any components you would like to use to display Discovery Search Results.
+2. Add `node-sass` as a dev dependency
```

Comment:
Is there a good way to automate this, as part of step (1)?

Reply:
i don't know of a good way to add dependencies and dev dependencies in the same command unless we make a special setup script (which can be covered in https://github.ibm.com/Watson-Discovery/disco-widgets/issues/436